src/facade/memcache_parser.cc

```diff
@@ -79,10 +79,19 @@ MP::Result ParseValueless(TokensView tokens, MP::Command* res) {
     }
     ++key_pos;
   }
-  res->key = tokens[key_pos++];
 
-  if (key_pos < num_tokens && base::_in(res->type, {MP::STATS, MP::FLUSHALL}))
-    return MP::PARSE_ERROR;  // we do not support additional arguments for now.
```

Comment:
The flow is broken, literally the line above conflicts with the if, we can't safely parse a key(!?) for STATS or FLUSHALL. I quick fixed it below